### ISSUE TITLE
Icons Request: Top Down flight Icons

### ISSUE THREAD

**Issue by spydmobile** [FortAwesome/Font-Awesome#13096]
Hi There, 
Any application of flight following will require top down profiles of aircraft so they can be turned to their heading. 
I would be happy with Drone, Helicopter and airplane, but an application of flight following would be better server by:
Fixed Wing Drone
Rotary Wing Drone
Small airplane (like a cessna)
med airplane (like a lear jet)
large airplane (like a Dash-8 Beech 19 , 747 etc)
Cargo Aircraft (C5 Galaxy or b52)

some top down suggestions were already offered in this ticket:
#2007

**Comment by anarchodin** [comment]
I'd like to point out that the icons plane, fighter-jet and space-shuttle are all already top-down views.

**Comment by spydmobile** [comment]
good point, but not many flight following applications homebrewed with FontAwesome will be tracking space shuttles and Fighter jets.

**Comment by anarchodin** [comment]
The point wasn't "we don't need these", it was "top-down views are dominant, the helicopter is the exception". :)

**Comment by spydmobile** [comment]
Ah I see I misunderstood, and I agree :)
I confess, I build live flight maps for wildfire and find it nuts that I have to create all my own icons and use FA everywhere else ;-)

**Comment by anarchodin** [comment]
What, specifically, is the violation? Those guidelines are rather on the vague side.
